approved/v93k/prb2.tf

```diff
@@ -190,16 +190,16 @@ test_flow
     }, open,"prb2_main", ""
     run(erase_all_2_814CEB0);
     run(margin_read1_all1_2_814CEB0);
+    if @EXTRA_TESTS == 1 then
     {
-      if @EXTRA_TESTS == 1 then
       {
         run(pgm_ckbd_1_814CEB0);
         run(mrd_ckbd_1_814CEB0);
-      }
-      else
-      {
-      }
-    }, open,"prb2_main_2", ""
+      }, open,"prb2_main_2", ""
+    }
+    else
+    {
+    }
```

Comment:
Logically the same, just that the group is now inside the flag gate rather than the other way around.
The new output matches the way it is defined in the source file.